apps/vaporgui/VLineEdit.h

```diff
@@ -0,0 +1,129 @@
+#pragma once
+
+#include <string>
+
+#include <QWidget>
+#include <QLabel>
+#include <QLineEdit>
+#include <QSpinBox>
+#include <QCheckBox>
+#include <QWidgetAction>
+
+#include "VContainer.h"
+
+//! class VLineEdit
+//! Wraps a QLineEdit with vaporgui's standard setter/getter functions.
+//! Handles string and double types, as well as precision and display of
+//! double values.
+
+class VLineEdit : public VContainer {
+    Q_OBJECT
+
+public:
+    VLineEdit( const std::string& value = "");
+
+    ~VLineEdit();
+
+    void SetValue( const std::string& value );
+    std::string GetValue() const;
+
+    void SetIsDouble( bool isDouble );
```

Comment:
What about integers? Also doesn't seem to use a validator.

Reply:
This is because I handle integer input with VSpinBoxes, not VLineEdits.  I could do this in a feature request, but it's not currently being used this way in the codebase.